net-misc/streamlink/streamlink-0.3.2.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_REQ_USE='xml(+),threads(+)'
+
+inherit distutils-r1
+
+DESCRIPTION="CLI for extracting streams from websites to a video player of your choice"
+HOMEPAGE="https://streamlink.github.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~arm ~x86"
```

Comment:
You have unkeyworded deps on ~arm, so you need to drop that kw. My suggestion would be to drop it here, leave the older ebuild alone if it still works (i.e. keep it) and we'll make a big keywording for pycryptodome and revdeps at some point.